src/test/HostActivationTests/GivenThatICareAboutStandaloneAppActivation.cs

```diff
@@ -50,6 +50,12 @@ public void Running_Build_Output_Standalone_EXE_with_DepsJson_and_RuntimeConfig_
 
             var appExe = fixture.TestProject.AppExe;
 
+            // TODO: Use FS.Chmod when build utility project is converted to csproj.
```

Comment:
Can you add a link to [https://github.com/NuGet/Home/issues/4424](https://github.com/NuGet/Home/issues/4424)?  That way we know what issue it is tracking.

---
In reply to: [114636019](https://github.com/dotnet/core-setup/pull/2259#discussion_r114636019) [](ancestors = 114636019)